Traceback (most recent call last):
  File "/home/bleyerj/.local/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/bleyerj/.local/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/bleyerj/.local/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/bleyerj/.local/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/bleyerj/.local/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/bleyerj/.local/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import matplotlib.pyplot as plt
import numpy as np
from mpi4py import MPI
import gmsh
import ufl
from dolfinx import fem, mesh, io, nls
from dolfinx.io.gmshio import model_to_mesh

import dolfinx.fem.petsc
import dolfinx.nls.petsc


# Define problem constants
Tref_value = fem.Constant(domain, 293.15)
DThole_value = 10.0
dt = fem.Constant(domain, 0.0)  # time step
------------------

----- stderr -----
hwloc/linux: Ignoring PCI device with non-16bit domain.
Pass --enable-32bits-pci-domain to configure to support such devices
(warning: it would break the library ABI, don't enable unless really needed).
hwloc/linux: Ignoring PCI device with non-16bit domain.
Pass --enable-32bits-pci-domain to configure to support such devices
(warning: it would break the library ABI, don't enable unless really needed).
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[1], line 14[0m
[1;32m     10[0m [38;5;28;01mimport[39;00m [38;5;21;01mdolfinx[39;00m[38;5;21;01m.[39;00m[38;5;21;01mnls[39;00m[38;5;21;01m.[39;00m[38;5;21;01mpetsc[39;00m
[1;32m     13[0m [38;5;66;03m# Define problem constants[39;00m
[0;32m---> 14[0m Tref_value [38;5;241m=[39m fem[38;5;241m.[39mConstant([43mdomain[49m, [38;5;241m293.15[39m)
[1;32m     15[0m DThole_value [38;5;241m=[39m [38;5;241m10.0[39m
[1;32m     16[0m dt [38;5;241m=[39m fem[38;5;241m.[39mConstant(domain, [38;5;241m0.0[39m)  [38;5;66;03m# time step[39;00m

[0;31mNameError[0m: name 'domain' is not defined

